protocol/0012-POSR-position_resolution.md

```diff
@@ -2,167 +2,82 @@
 
-}
+- If a party marked as "distressed" has the cross-margin mode selected for the market being considered:
 
-```
+  - all their open orders on that market get cancelled without releasing any collateral from the margin account,
+  - their margin level then gets recalculated to account only for the open volume they hold in a market
+  - if party's margin account balance is now above or equal to their latest maintenance margin level then such party is no longer marked as "distressed".
 
-#### STEP 4
+- If party's is marked as "distressed" at this stage then:
 
-Close out trades are generated with the distressed traders
+  - their open volume gets added to network's open volume for that market,
+  - party's open volume in that market gets set to 0,
+  - the full amount of party's margin account balance gets transferred to market's insurance pool.
 
-```json
-CloseOutTrade1 {
-  buyer: Network,
-  seller: Trader1,
-  size: 5,
-  price: 113.33,
-  type: 'safety-provision'
-}
+This concludes the position resolution from party's perspective.
 
-CloseOutTrade2 {
-  buyer: Trade2,
-  seller: Network,
-  size: 4,
-  price: 113.33,
-  type: 'safety-provision'
-}
+#### Managing network's position
 
-CloseOutTrade3 {
-  buyer: Network,
-  seller: Trader3,
-  size: 2,
-  price: 113.33,
-  type: 'safety-provision'
-}
-```
+Whilst network has a non-zero position in a given market it's treated as any other party with market's insurance in that market acting as its margin account and with and exception that margin search, margin release or liquidation are never attempted on a network party.
 
-This results in the open position sizes for all distressed traders and the network entities to be zero.
+Whenever the network party has a non-zero position it attempts to unload it using an [immediate or cancel](./0014-ORDT-order_types.md) limit order. If the the network has a `long` position it will submit a `sell` order. If it has a short position it will submit a buy order. The size of that order is chosen according to the liquidation strategy which forms a part of the market's configuration. The strategy can be updated at any point whilst market is active with a market change [governance vote](./0028-GOVE-governance.md#2-change-market-parameters).
 
-`// OpenPosition of Network =  -3 +5 -4 +2 = 0`
-`// OpenPosition of Trader1 =  +5 -5 = 0`
-`// OpenPosition of Trader2 = -4 +4 =  0`
-`// OpenPosition of Trader3 =  +2 - 2 = 0`
+Currently only one liquidation strategy is supported and its defined by the following parameters:
 
-#### STEP 5
+- `disposal time step` (min: `1s`, max: `1h`, default: `n=10s`): network attempts to unload its position in a given market every time it goes out of auction and then every `n` seconds as long as market is not in auction mode and while the network's position is not equal to `0`,
```

Comment:
Seeing as these are parameters that can be specific to each market, but we want to have default values, should the defaults be hard-coded, or set as network parameters? Can those network parameters be changed via governance, or not?

Reply:
Let's hardcore them, the netparam default for a market parameter ends up just confusing everyone. I'll mention that in the spec.

Reply:
@barnabee wonderiing what you make of default values in proposals here

Reply:
I'd suggest no defaults. Make them specify it on the proposal.

Reply:
what about market already existing after a protocol upgrade?

Reply:
Defaults in protocol upgrade are fine, but that distinction should be made clearer probably -> proposals _must_ specify all 4 parameters, existing markets will default to these default params, which can be updated through governance after protocol upgrade

Reply:
OK, let me know how it's looking now.